libs/server/features/src/security-pricing/security-pricing.service.ts

```diff
@@ -89,22 +91,24 @@ export class SecurityPricingService implements ISecurityPricingService {
                     take: count,
                 }),
         })) {
-            const livePricing = await this.marketDataService
-                .getLivePricing(securities)
-                .then((lp) => lp.filter((p) => !!p.pricing))
+            const pricingData =
+                process.env.NX_POLYGON_TIER === 'basic'
+                    ? await this.marketDataService.getEndOfDayPricing(securities, allPrices)
+                    : await this.marketDataService.getLivePricing(securities)
+            const prices = pricingData.filter((p) => !!p.pricing)
```

Comment:
Error handling for all `await` statements would be very useful here. Currently whenever the job fails, I don't get an error message.